contrib/java/KalmanFilter.java

```diff
@@ -87,4 +85,18 @@ public void setMeasurementNoise(double noise){
 	public void setProcessNoise(double noise){
 		    this.R = noise;
 	}
+    
+    /**
+      * Main method for usage example
+      *
+      */
+    public static void main(String[] args){
```

Comment:
Please remove this method. Examples can be placed in a README.md but should not be present in the core class.